composer.json

```diff
@@ -37,7 +37,7 @@
     "typo3/cms-tstemplate": "^10.4.37|^11.5.33",
     "caseyamcl/phpoaipmh": "^3.3.1",
     "ubl/php-iiif-prezi-reader": "0.3.0",
-    "solarium/solarium": "^5.2.0"
+    "solarium/solarium": "5.2.0 - 6.0.99"
```

Comment:
```suggestion
    "solarium/solarium": "5.2.0 - 6.3"
```

Reply:
Solarium 6.3.5 seems to work as good as 6.0.x (most CI tests pass, one error for OAI).